docs/openstack/ceph_backend_configuration.md

```diff
@@ -19,7 +19,18 @@ CEPH_KEY=$($CEPH_SSH "cat /etc/ceph/ceph.client.openstack.keyring | base64 -w 0"
 CEPH_CONF=$($CEPH_SSH "cat /etc/ceph/ceph.conf | base64 -w 0")
 ```
 
-## Procedure - Ceph backend configuration
+## Modify capabilities of the "openstack" user to accommodate Manila
+
+On TripleO environments, the CephFS driver in Manila is configured to use 
```

Comment:
should we mention the reason behind this choice? 
- we have the new driver that doesn't require a separated keyring because of the caps
- we already use a single keyring for all the openstack services and in a greenfield deployment Manila uses the same approach (hence we're basically converging to the same pattern adopted for fresh deployments)

Reply:
Done